packages/web-runtime/src/pages/account.vue

```diff
@@ -160,14 +161,19 @@ export default defineComponent({
 
     // FIXME: Use settings service capability when we have it
     const isSettingsServiceSupported = useCapabilitySpacesEnabled()
-    // FIXME: Use change password capability when we have it
-    const isChangePasswordEnabled = useCapabilitySpacesEnabled()
     const spacesEnabled = useCapabilitySpacesEnabled()
     const isPersonalDataExportEnabled = useCapabilityGraphPersonalDataExport()
+
     const user = computed(() => {
       return store.getters.user
     })
 
+    const isChangePasswordDisabled = computed(() => {
```

Comment:
Please changeb ack to only use `useCapabilityChangeSelfPasswordDisabled()` (also see comment above for the default of the capability fallback)